lars-reimann/api-editor
# fix: style math blocks appropriately

Closes #131.

## Summary of Changes
Previously only inline math formulae where rendered properly. This was now extended to block formulae as well.

## Screenshots (if necessary)

The comparison was made for [sklearn / sklearn.metrics._classification / cohen_kappa_score](https://lars-reimann.github.io/api-editor/#/sklearn/sklearn.metrics._classification/cohen_kappa_score).

### Before
![image](https://user-images.githubusercontent.com/2501322/126547019-13536d6c-64be-415c-9d2e-6632d3d1b9c6.png)


### After
![image](https://user-images.githubusercontent.com/2501322/126547093-43bdd64a-43fd-40ae-bef3-182141ec7ae4.png)


## Testing instructions
Ensure block formulae are rendered as expected (as long as the LaTeX in the documentation of scikit-learn is correct):
1. Search for `.. math::` in the JSON to find such cases. 
2. Compare with the scikit-learn documentation, e.g. for [sklearn.metrics.cohen_kappa_score](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.cohen_kappa_score.html).
